bitex/interface/base.py

```diff
@@ -52,6 +66,21 @@ def _get_supported_pairs(self):
         """
         raise NotImplementedError
 
+    def _get_supported_pairs_formatted(self):
+        """Return a list of supported pairs."""
+        pairs = self._supported_pairs
+        pairs_formatted = []
+        for pair in pairs:
+            pair = pair.upper()
+            if '_' in pair:
+                pairs_formatted.append(pair)
+            else:
+                if pair[-4:] == 'USDT':
+                    pairs_formatted.append(pair[:-4] + '_USDT')
+                else:
+                    pairs_formatted.append(pair[:-3] + '_' + pair[-3:])
+        return pairs_formatted
+
```

Comment:
no formatting of exchange symbols that are fetched from the exchange - this is what we check against. If anything, we must ensure our formatters match the symbols returned from `_get_supported_pairs`